### ISSUE TITLE
a small question about Deepglobe datase

### ISSUE THREAD

**Issue by Jack-bo1220** [cedricgsh/ISDNet#2]
Hi,
First of all, thank you for your nice work. However, I have a small question about Deepglobe dataset.

As far as I know, the original validation and test sets do not provide labels. Are all the val and test sets in the current data split from the original training set?
According to your and GLNet's split file (.txt), the total number of train samples is 454 rather than 455. Why do you and your previous papers on related work explain that 455？

Thank you for your attention!

**Comment by cedricgsh** [comment]
@Jack-bo1220 Hi, thank you for interests in our work. 
1)We follow the setting of GLNet to split train, val and test sets. 
2)Thank you for point out this question. We do neglect the number of train samples. FCtL and MagNet all explain that 455. Maybe you can ask the author of GLNet, they first split the DeepGlobe datasets

**Comment by Jack-bo1220** [comment]
thanks very much. Look forward to seeing your open code soon. @cedricgsh
